radixdlt/src/main/java/com/radixdlt/consensus/sync/SyncManager.java

```diff
@@ -0,0 +1,150 @@
+package com.radixdlt.consensus.sync;
```

Comment:
Missing header comment.  See StateSyncNetwork.java for how this should look.

Reply:
Done